### PR TITLE
Cleanup version and setup code, add changelog, increase version number

### LINKED ISSUE #299: Update version number.
We should probably update the version:  (a) it has come a long way since we've last done it (b) it is helpful if the version number changes so that you can check.

### PR DESCRIPTION
**Pull request by bhazelton**
fixes #299

### CODE REVIEW

**Comment by bhazelton** [comment]
@ddeboer turns out yaml is only needed for testing, not for running the code. So I'll add it a slightly different way to setup.py

File: hera_mc/__init__.py [hunk 1]
```diff
@@ -84,7 +84,9 @@ def NotNull(kind, **kwargs):
     return Column(kind, nullable=False, **kwargs)
 
 
-from .version import __version__  # noqa
+from . import version  # noqa
+__version__ = version.version
```
Comment (plaplant) [inline hera_mc/__init__.py]: Is there a reason why we have to define this here, and we can't wait until all the imports are done?

Comment (bhazelton) [reply hera_mc/__init__.py]: This is the only place the `__version__` attribute is defined. Oh! you mean move that line after the imports. Sure, I can do that.

**Comment by bhazelton** [comment]
@plaplant I don't think we can, there's a bunch of other code above those imports that can't be moved after them

**Pull request merged**